api/db/changelog/db.changelog-251-add-workspace_inaccessible_to_sa.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+  <changeSet author="thibault" id="changelog-251-add-workspace_inaccessible_to_sa">
+    <createTable tableName="workspace_inaccessible_to_sa">
```

Comment:
new table to indicate that this workspace is inaccessible to the GAE SA in Terra, despite being marked as "active" in the RWB DB.

All normal active workspaces should be accessible to the GAE SA in Terra.  This indicates a problem of some kind.  Many workspaces on Test are like this, particularly older workspaces.  I would not expect to see many of these on Prod / higher environments, but this cron will reveal any which do exist by adding them to this table.

Open to naming suggestions.